packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -945,10 +949,26 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
   }
 
   // Returns the amount of delta that was not used.
+  //
+  // Positive delta means going down (exposing stuff above), negative delta
+  // going up (exposing stuff below).
   double applyClampedDragUpdate(double delta) {
     assert(delta != 0.0);
-    final double min = delta < 0.0 ? -double.INFINITY : minScrollExtent;
-    final double max = delta > 0.0 ? double.INFINITY : maxScrollExtent;
+    // If we are going towards the maxScrollExtent (negative scroll offset),
```

Comment:
The first sentence here is not adding providing more information than reading the one line of code.

What I would find useful here is a comment saying, how we get to a state where pixels is out of the min/max scroll extent, and why are we not clamping in this case.

Reply:
added a paragraph that explains that